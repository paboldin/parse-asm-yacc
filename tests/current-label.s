	.data
abc: def: ghj: nop;
