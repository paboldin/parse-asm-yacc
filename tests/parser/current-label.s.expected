(DIRECTIVE_DATA)	.data
(LABEL)abc(LABEL) def(LABEL) ghj
(TOKEN) nop
(LABEL)abc(LABEL) def(LABEL) ghj
symbol: name = ;, type = unknown
symbol: section = .data
symbol: label = (l4)(LABEL)abc(LABEL) def(LABEL) ghj
(l4)(LABEL)abc(LABEL) def(LABEL) ghj
symbol: name = ghj, type = unknown
symbol: section = .data
symbol: label = (l2)(LABEL)abc(LABEL) def(LABEL) ghj
(l2)(LABEL)abc(LABEL) def(LABEL) ghj
(l2)(TOKEN) nop
section: name = .data, flags = 
(l1)(DIRECTIVE_DATA)	.data
section: name = .text, flags = x
	.data
abc: def: ghj: nop;

abc: def: ghj:
